errors/apiErrors.ts

```diff
@@ -238,17 +272,17 @@ export function getAxiosErrorWithContext(
  * @throws
  */
 export function throwApiError(
-  error: AxiosError,
+  error: HubSpotHttpError,
   context: AxiosErrorContext = {}
 ): never {
   if (error.isAxiosError) {
```

Comment:
I can't remember why we originally included this `isAxiosError` check. Do we still need it?

Reply:
Probably not, I'm going to work on the integration in tandem with this and start to remove some of the backwards compatibility holdovers.